### PR TITLE
bug fixing gearmand crash on logging worker's exception request

### PR DESCRIPTION
**Pull request by p-alik**
an alternative for PR https://github.com/gearman/gearmand/issues/169
to solve https://github.com/gearman/gearmand/issues/168

### CODE REVIEW

**Comment by esabol** [comment]
LGTM. Please merge.

**Comment by esabol** [comment]
"gearmand" is misspelled in the PR title. I only mention this because I think the misspelling may end up in the release notes due to the way `git log` is used in the release process. You might want to fix that prior to merging.

**Comment by p-alik** [comment]
thank you for mentioning it. Misspelling is corrected in PR title and commit message.

**Comment by esabol** [comment]
Um, there's still a stray single quote after "gearmand" in the PR title.

File: libgearman-server/server.cc [hunk 1]
```diff
@@ -807,15 +807,16 @@ gearmand_error_t gearman_server_run_command(gearman_server_con_st *server_con,
...
                                                                 (char *)(packet->arg[0]), (size_t)strlen(packet->arg[0]),
                                                                 server_con);
-      gearmand_log_debug(GEARMAN_DEFAULT_LOG_PARAM,
-                         "Exception being sent from: %.*s(%lu)",
-                         server_job->function->function_name_size, server_job->function->function_name, server_job->function->function_name_size);
       if (server_job == NULL)
       {
         return _server_error_packet(GEARMAN_DEFAULT_LOG_PARAM, server_con, GEARMAN_JOB_NOT_FOUND, 
-                                    gearman_literal_param("An exception was received for a job that does not exist"));
+                                  gearman_literal_param("An exception was received for a job that does not exist"));
```
Comment (esabol) [inline libgearman-server/server.cc]: Is this whitespace change intentional?

Comment (p-alik) [reply libgearman-server/server.cc]: No. The change is revoked https://github.com/gearman/gearmand/pull/183/commits/991db14972411b2288e90b80c725a74b24c47624

**Comment by esabol** [comment]
Looks good to me. Please merge!

**Pull request merged**